src/components/institution_table/InstitutionTable.tsx

```diff
@@ -0,0 +1,90 @@
+import React, {useEffect, useState} from 'react';
+import {isEmpty} from 'lodash';
+import ReactTooltip from 'react-tooltip';
+import PaginationBar from 'src/components/PaginationBar';
+import SimpleTable from 'src/components/SimpleTable';
+import {Storage} from 'src/libs/storage';
+import {
+  calcPageCount,
+  CellData,
+  columnHeaderData,
+  columns,
+  getInitialSort,
+  processRows,
+  SortType,
+  storageInstitutionSort,
+  tableStyles
+} from 'src/components/institution_table/InstitutionTableUtils';
+import {Institution} from 'src/types/model';
+import {recalculateVisibleTable} from 'src/libs/utils';
+
+export interface InstitutionTableProps {
+  readonly filteredList: Institution[];
+  readonly currentPage: number;
+  readonly setCurrentPage: (page: number) => void;
+  readonly tableSize: number;
+  readonly setTableSize: (size: number) => void;
+}
+
+export default function InstitutionTable(props: InstitutionTableProps) {
+  const {filteredList, currentPage, setCurrentPage, tableSize, setTableSize} = props;
+  const [pageCount, setPageCount] = useState<number>(calcPageCount(tableSize, filteredList));
+  const [visibleRows, setVisibleRows] = useState<CellData[][]>([]);
+  const [sort, setSort] = useState<SortType>(getInitialSort(columns));
+
+  useEffect(() => {
+    recalculateVisibleTable({
+      tableSize,
+      pageCount,
+      filteredList: processRows(filteredList),
+      currentPage,
+      setPageCount,
+      setCurrentPage,
+      setVisibleList: setVisibleRows,
+      sort
+    });
+    ReactTooltip.rebuild();
+  }, [tableSize, pageCount, filteredList, currentPage, setPageCount, setCurrentPage, sort]);
+
+  const changeTableSize = (newTableSize: number) => {
+    if (!isEmpty(newTableSize) && newTableSize > 0) {
+      setTableSize(newTableSize);
+    }
+  };
+
+  const goToPage = (page: number) => {
+    if (page > 0 && page < pageCount + 1) {
+      setCurrentPage(page);
+    }
+  };
+
+  return (
+      <div data-cy="institution-table">
+        <SimpleTable
```

Comment:
Using this component is the meat of the change. Originally, we were writing out table elements directly. This change allows us to use the same styling (e.g. material-ui) as the rest of our console pages.